_drafts/oryx/versioning.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Versioning
+description: Versioning
+template: concept-topic-template
+---
+
+# Versioning
+In this document, we outline the methods we use to deliver an advanced application development platform while maintaining stability. Our goal is to ensure that any upcoming changes are introduced in a predictable manner. We want all Oryx users to be informed about the addition of new features and adequately prepared for the removal of outdated ones.
+
+Stability is a critical aspect of any software framework, and it's especially important for frontend frameworks like Oryx. The stability of a framework refers to its ability to function reliably and consistently over time, despite changes in the environment or updates to its components. 
+
+Ensuring that a framework is stable requires careful planning, testing, and versioning. In this article, we will explore how the Oryx framework achieves stability through its versioning and release processes. We'll look at how Oryx manages its releases, how it uses version numbers to indicate changes, and how it ensures backward compatibility for its users. 
+
+## Semantic versioning
+Oryx follows [semantic versioning](https://semver.org/) principles. 
+
+This indicates that with a version number x.y.z:
+- If there are critical bug fixes to be released, a patch release is made by altering the z number (e.g., 1.5.2 to 1.5.3).
+- If new features or non-critical fixes are being released, a minor release is made by modifying the y number (e.g., 1.5.2 to 15.6.0).
+- If there are significant changes that may break compatibility, a major release is made by changing the x number (e.g., 1.5.2 to 2.0.0).
+
+### Preview releases
+
+To give you a sneak peek of what's to come, we offer pre-release versions called Release Candidates (rc) for every major and minor release.
+
+## Release Process
+The Oryx framework comprises two distinct components, each of which is released separately:
+- Libraries
+- Labs
+
+### Libraries
+The primary sources of functionality within the Oryx frameworks are the Oryx libraries, which include core features, utilities and helpers, domain-specific functionality, as well as pre-designed themes and presets that can be readily used. 
+All the [packages](https://www.npmjs.com/org/spryker-oryx) released together under the same version. 
+
+### Labs package
```

Comment:
Is the `labs package` an uncommon name? 

The libraries are also delivered in packages, but the previous section is called just `Librarries`, not `Labraries package`.